src/Raven.Server/Config/Categories/LicenseConfiguration.cs

```diff
@@ -46,5 +46,16 @@ public class LicenseConfiguration : ConfigurationCategory
         [DefaultValue(false)]
         [ConfigurationEntry("License.SkipLeasingErrorsLogging", ConfigurationEntryScope.ServerWideOnly)]
         public bool SkipLeasingErrorsLogging { get; set; }
+        
+        
+        [Description("EXPERT ONLY. Disable automatic updating of the license")]
+        [DefaultValue(false)]
+        [ConfigurationEntry("License.DisableAutoLicenceUpdate", ConfigurationEntryScope.ServerWideOnly)]
+        public bool DisableAutoLicenceUpdate { get; set; }
+        
+        [Description("EXPERT ONLY. Disable checking the support options for the license")]
+        [DefaultValue(false)]
+        [ConfigurationEntry("License.DisableLicenseSupportMode", ConfigurationEntryScope.ServerWideOnly)]
```

Comment:
Can we change that to `License.DisableLicenseSupportCheck` ?

Reply:
done